src/pages/DetailPage/Components/CartButton.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import './CartButton.scss';
+
+class CartButton extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const titleData = this.props.mealKitInfoList;
```

Comment:
```suggestion
    const {stock} = this.props.mealKitInfoList;
```
바로 구조분해 해서 사용해도 되보입니다!